arch/arm/src/imxrt/imxrt_flexcan.c

```diff
@@ -1455,7 +1455,7 @@ static int imxrt_ioctl(struct net_driver_s *dev, int cmd,
       (FAR struct imxrt_driver_s *)dev->d_private;
 
   int ret;
-
+  struct flexcan_timeseg data_timing;
```

Comment:
Please move this struct declaration to above "int ret"

Reply:
@acassis No, this is for another project. But this definitely is a nice target for these kind of applications!